webpack.config.js

```diff
@@ -76,5 +72,8 @@ module.exports = {
       amd: 'hls.js',
       root: 'Hls'
     }
+  },
+  optimization: {
+    minimize: PROD
```

Comment:
use `--mode production`/`--mode development` in package.json instead(for build/dev/watch scripts)

Reply:
done